lib/index.js

```diff
@@ -55,6 +56,55 @@ exports.plugin = {
             server.ext('onPreStart', internals.initialize);
             server.ext('onPostStop', internals.stop);
 
+            // Expose an hpal command for schwifty-migration
+            server.expose('commands', {
+                'migrate:diff': (initializedServer, args, rootDir, ctx) => {
```

Comment:
Can you pull this into `internals` the way the decoration implementations, etc. also are?

Reply:
And in turn, no need to call it `initializedServer`—something short like `srv` or `server` should be ok!

Reply:
Yes